src/Qowaiv.DomainModel/AggregateRoot_TAggregate.cs

```diff
@@ -46,6 +46,12 @@ protected AggregateRoot(IValidator<TAggregate> validator)
         /// </remarks>
         protected abstract void AddEventsToBuffer(IEnumerable<object> events);
 
+        /// <summary>Clones the current instance.</summary>
+        /// <remarks>
+        /// It is advised to do this be replaying all previous events.
```

Comment:
You should clone the aggregate by replaying all events that led to the current state of affairs. Do you have a suggestion on how it could be explained better?